main.py

```diff
@@ -135,25 +122,26 @@ def add_new_stream(self, *args, stream_url=None, stream_quality=cfg[CONFIG_QUALI
         # Give some feedback to the user
         self.show_loading_gif()
 
-        # Run the rest on a seperate thread to be able to show the loading feedback
+        # Run the rest on a separate thread to be able to show the loading feedback
         # Also helps a lot with lag
         threading.Thread(target=self._add_new_stream, args=(stream_url, stream_quality)).start()
 
     def _add_new_stream(self, stream_url=None, stream_quality=cfg[CONFIG_QUALITY]):
         """Fetches qualities and if possible adds a frame to the main window."""
         try:
-            stream_options = self.streamlink_session.streams(stream_url)
+            stream_options = self.model.get_stream_options(stream_url)
 
             # If the stream is not currently broadcasting, 'stream_options'
             # will be an empty list.
             if not stream_options:
                 raise streamlink.exceptions.NoStreamsError(stream_url)
 
             if stream_quality not in stream_options:
-                self.fail_add_stream.emit(AddStreamError.DEFAULT_QUALITY_MISSING, (stream_options, stream_url, stream_quality))
+                self.fail_add_stream.emit(AddStreamError.DEFAULT_QUALITY_MISSING,
+                                          (stream_options, stream_url, stream_quality))
```

Comment:
Could we have this as: 
```python
self.fail_add_stream.emit(
    AddStreamError.DEFAULT_QUALITY_MISSING,
    (stream_options, stream_url, stream_quality)
)
```
instead? Or perhaps even:

```python
self.fail_add_stream.emit(
    AddStreamError.DEFAULT_QUALITY_MISSING,
    (
        stream_options, 
        stream_url, 
        stream_quality
    )
)
```

But you're the PEP8 master here so your call ❤️